sunraster/spectrogram.py

```diff
@@ -49,7 +50,59 @@
 SUPPORTED_EXPOSURE_NAMES = np.array(SUPPORTED_EXPOSURE_NAMES)
 
 
-class Raster(NDCube):
+class SpectrogramABC(abc.ABC):
+
```

Comment:
maybe add a docstring here just to help describe this a bit better?:)

Reply:
Dont wan't to end up in any confusing docstring inheritance situations.  ```NDCubeABC``` doesn't have a docstring.  I'll put a comment above the class instead explain what it is.